device-types/Cisco/MDS-C9148-16P.yaml

```diff
@@ -0,0 +1,137 @@
+---
+manufacturer: Cisco
+model: MDS 9148 16P
+slug: ds-9148-16p
```

Comment:
The slug should have `cisco-` prepended to it

Reply:
The slug has been prepended with "cisco-".